src/index.js

```diff
@@ -54,11 +61,21 @@ const unbindEvents = function CozyBarUnbindEvents () {
 }
 
 const getDefaultStackURL = function GetDefaultCozyURL () {
-  return document.querySelector('[role=application]').dataset.cozyDomain
+  const appNode = document.querySelector(APP_SELECTOR)
+  if (!appNode) {
+    console.warn(`Cozy-bar is looking for a "${APP_SELECTOR}" tag that contains your application and can't find it :'(… The BAR is now disabled`)
+    return ''
+  }
+  return appNode.dataset.cozyDomain
 }
 
 const getDefaultToken = function GetDefaultToken () {
-  return document.querySelector('[role=application]').dataset.cozyToken
+  const appNode = document.querySelector(APP_SELECTOR)
+  if (!appNode) {
+    console.warn(`Cozy-bar is looking for a "${APP_SELECTOR}" tag that contains your application and can't find it :'(… The BAR is now disabled`)
```

Comment:
Like previous comment, we may adapt the warning message to "Cozy-bar can't discover the app's token, and will probably fail to initialize the connection with the stack"